lib/python/satos_payload_sdk/antaris_file_download.py

```diff
@@ -1,43 +1,42 @@
-import logging
-
 from azure.storage.fileshare import ShareFileClient
+import logging
 
-from satos_payload_sdk import antaris_api_common as api_common
-
-
-logger = logging.getLogger("satos_payload_sdk")
-
-azure_logger = logging.getLogger("azure.core.pipeline.policies.http_logging_policy")
-azure_logger.setLevel(logging.WARNING)
-
+logger = logging.getLogger("azure.core.pipeline.policies.http_logging_policy")
+logger.setLevel(logging.WARNING)
 
 g_FTM = "FTM"
 g_File_String = "File_Conn_Str"
 g_Truetwin_Dir = "Truetwin_Dir"
 g_Share_Name = "Share_Name"
 g_Outbound_Path_Prefix = "/opt/antaris/outbound/"
 
-
 class File_Stage():
 
-    def __init__(self, download_file_params, config_data):
-        self.config_data = config_data
-        self.file_path_local = g_Outbound_Path_Prefix + self.download_file_params.file_path
-        self.file_path_remote = self.config_data[g_FTM][g_Truetwin_Dir] + "/" + self.file_name
-
-    def file_download(self):
-        cl = ShareFileClient.from_connection_string(
-                conn_str=self.config_data[g_FTM][g_File_String],
-                share_name=self.config_data[g_FTM][g_Share_Name],
-                file_path=self.file_path_remote
-        )
+    def __init__(self, download_file_params,config_data ): 
```

Comment:
Do we need to change anything in python application to make the file transfer compatible with changes made here in library functions?

Reply:
File transfer function is moved out of class. We are not creating any class object and hence no change in class needed.